internal/driver/cli.go

```diff
@@ -140,12 +142,30 @@ func parseFlags(o *plugin.Options) (*source, []string, error) {
 		Comment:      *flagAddComment,
 	}
 
+	var base []string
```

Comment:
This fragment of code of dealing with base and diffBase flags is a bit long now. Consider refactoring it.

Reply:
Did some refactoring.